Lib/test/test_peepholer.py

```diff
@@ -335,6 +422,33 @@ def forloop():
                 pass
         self.assertEqual(count_instr_recursively(forloop, 'BUILD_LIST'), 0)
 
+    def test_multiline_statements_are_not_treated_differently(self):
```

Comment:
This test is no longer needed. These cases are covered by other tests (the first case is equal to test_elim_jump_to_uncond_jump2, and the second case is identical to test_elim_jump_to_uncond_jump).